bindata/network/multus/multus.yaml

```diff
@@ -455,3 +455,42 @@ spec:
           configMap:
             name: cni-binary-copy-script
             defaultMode: 0744
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: ip-reconciler
+  namespace: openshift-multus
+  labels:
+    tier: node
+    app: whereabouts
+spec:
+  schedule: "*/5 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          priorityClassName: "system-node-critical"
```

Comment:
This seems maybe incorrect for this job? I forget exactly what this field does, but it seems like a periodic cleanup job is not "system critical".

Reply:
As I said, I don't remember where the exact definition of how to deal with this field is... (maybe something in openshift/enhancements/dev-guides says? If not, probably there _should_ be something there).

But, for example, if a node is at max capacity, and it's time to run this job, do you want kubernetes to kill user workload pods to make room for this pod? I'm guessing no, in which case this probably needs to have different priority characteristics than most other network-related pods.